File: CodeForces/Python/CF_791A_BearAndBigBrother.py
# Problem: 791A. Bear and big brother
# Link: https://codeforces.com/problemset/problem/791/A

limak, bob = map(int, input().split())

years = 0
while limak <= bob:
    limak *= 3
    bob *= 2
    years += 1

print(years)

File: CodeForces/Java/CF_791A_BearAndBigBrother.java
//  Problem: 791A. Bear and big brother
//  Link: https://codeforces.com/problemset/problem/791/A

import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    Scanner scan = new Scanner(System.in);
    
    int limak = scan.nextInt();
    int bob = scan.nextInt();
    scan.close();
    
    int years = 0;

    while (limak <= bob) {
        limak *= 3;
        bob *= 2;
        years++;
    }
        System.out.println(years);
  }
}
